doc/changes/changes_2.0.1.md

```diff
@@ -0,0 +1,22 @@
+#Virtual Schema for BigQuery 2.0.1, released 2021-11-23
+
+Code name: Dependency Updates
```

Comment:
Do you want to add a version here?

Reply:
no. I'll add it only to the GitHub release. Here it would be doubled.